spec/polaris-management-service.yml

```diff
@@ -1136,6 +1136,11 @@ components:
                 for accessing data and metadata files within the related catalog. Setting this property to `true`
                 effectively disables vending storage credentials to clients. This setting is intended for configuring
                 catalogs with S3-compatible storage implementations that do not support STS.
+            userTokenSts:
```

Comment:
suggestion: `propagateApiUserIdentity` - it is more directly connected to actual code behaviour... at least for me 😅 I'd welcome other naming suggestions too. Better have this discussion now than after the stuff is released :)

Reply:
Discussion happening in the other thread (https://github.com/apache/polaris/pull/3170#discussion_r2566597009), will close this comment